src/classes/ADDR_IValidator.cls

```diff
@@ -35,19 +35,43 @@
 */
 public Interface ADDR_IValidator {
 
-    /** Return the service name to be displayed in the drop-down list. **/
+    /** 
+    * @description returns the service name to be displayed in the drop-down list. 
+    * @return string
+    */
     String getServiceName();
     
-    /** Return the default endpoint URL. **/
+    /** 
+    * @description returns the default endpoint URL. 
+    * @return string
+    */
     String getDefaultURL();
     
-    /** Return true if batch address validation is supported. If not, the batch address validation button
-    will be disabled. **/
+    /** 
+    * @description returns the help text to be shown when the user selects the service from the drop-down. It
+    * may specify things such as the required fields. 
+    * @return string
+    */
+    String getHelpText();
```

Comment:
Here, this is new - so it need to be implemented by everything that implements this ADDR_IValidator interface